`FocusableWrapper` removed from upstream gpui library which is breaking on update

Getting below error while compiling the application:

```shell
error[E0432]: unresolved import `gpui::FocusableWrapper`
 --> /Users/nilesh/.cargo/git/checkouts/gpui-component-95ce574d8a0da8b8/646b8b7/crates/ui/src/event.rs:1:29
  |
1 | use gpui::{App, ClickEvent, FocusableWrapper, InteractiveElement, Stateful, Window};
  |                             ^^^^^^^^^^^^^^^^ no `FocusableWrapper` in the root

error[E0432]: unresolved import `gpui::FocusableWrapper`
 --> /Users/nilesh/.cargo/git/checkouts/gpui-component-95ce574d8a0da8b8/646b8b7/crates/ui/src/menu/context_menu.rs:5:81
  |
5 |     Corner, DismissEvent, DispatchPhase, Element, ElementId, Entity, Focusable, FocusableWrapper,
  |                                                                                 ^^^^^^^^^^^^^^^^ no `FocusableWrapper` in the root

```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -26,6 +26,7 @@ jobs:
     runs-on: ${{ matrix.run_on }}
     steps:
       - uses: actions/checkout@v4
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
       - name: Install system dependencies
         if: ${{ matrix.run_on != 'windows-latest' }}
         run: script/bootstrap

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -22,9 +22,14 @@ wef = { path = "crates/wef" }
 
 anyhow = "1"
 log = "0.4"
-serde = "1.0.203"
+serde = { version = "1.0.219", features = ["derive"] }
 serde_json = "1"
+schemars = "1"
 smallvec = "1"
+rust-i18n = "3"
+raw-window-handle = "0.6.2"
+smol = "1"
+tracing = "0.1.41"
 
 [workspace.dependencies.windows]
 features = ["Wdk", "Wdk_System", "Wdk_System_SystemServices"]

--- a/crates/ui/Cargo.toml
+++ b/crates/ui/Cargo.toml
@@ -19,7 +19,15 @@ webview = ["dep:wry"]
 [dependencies]
 gpui.workspace = true
 gpui-component-macros.workspace = true
-anyhow = "1"
+rust-i18n.workspace = true
+schemars.workspace = true
+serde.workspace = true
+serde_json.workspace = true
+smallvec.workspace = true
+smol.workspace = true
+tracing.workspace = true
+anyhow.workspace = true
+
 enum-iterator = "2.1.0"
 futures-util = "0.3.31"
 image = "0.25.1"
@@ -28,13 +36,6 @@ once_cell = "1.19.0"
 paste = "1"
 regex = "1"
 resvg = { version = "0.45.0", default-features = false, features = ["text"] }
-rust-i18n = "3"
-schemars = "0.8.22"
-serde = "1.0.203"
-serde_json = "1"
-smallvec = "1.13.2"
-smol = "1"
-tracing = "0.1.41"
 unicode-segmentation = "1.12.0"
 usvg = { version = "0.45.0", default-features = false, features = ["text"] }
 uuid = "1.10"

--- a/crates/ui/src/event.rs
+++ b/crates/ui/src/event.rs
@@ -1,4 +1,4 @@
-use gpui::{App, ClickEvent, FocusableWrapper, InteractiveElement, Stateful, Window};
+use gpui::{App, ClickEvent, InteractiveElement, Stateful, Window};
 
 pub trait InteractiveElementExt: InteractiveElement {
     /// Set the listener for a double click event.
@@ -18,5 +18,4 @@ pub trait InteractiveElementExt: InteractiveElement {
     }
 }
 
-impl<E: InteractiveElement> InteractiveElementExt for FocusableWrapper<E> {}
 impl<E: InteractiveElement> InteractiveElementExt for Stateful<E> {}

--- a/crates/ui/src/menu/context_menu.rs
+++ b/crates/ui/src/menu/context_menu.rs
@@ -2,9 +2,9 @@ use std::{cell::RefCell, rc::Rc};
 
 use gpui::{
     anchored, deferred, div, prelude::FluentBuilder, px, relative, AnyElement, App, Context,
-    Corner, DismissEvent, DispatchPhase, Element, ElementId, Entity, Focusable, FocusableWrapper,
-    GlobalElementId, InspectorElementId, InteractiveElement, IntoElement, MouseButton,
-    MouseDownEvent, ParentElement, Pixels, Point, Position, Stateful, Style, Window,
+    Corner, DismissEvent, DispatchPhase, Element, ElementId, Entity, Focusable, GlobalElementId,
+    InspectorElementId, InteractiveElement, IntoElement, MouseButton, MouseDownEvent,
+    ParentElement, Pixels, Point, Position, Stateful, Style, Window,
 };
 
 use crate::menu::popup_menu::PopupMenu;
@@ -19,7 +19,6 @@ pub trait ContextMenuExt: ParentElement + Sized {
 }
 
 impl<E> ContextMenuExt for Stateful<E> where E: ParentElement {}
-impl<E> ContextMenuExt for FocusableWrapper<E> where E: ParentElement {}
 
 /// A context menu that can be shown on right-click.
 pub struct ContextMenu {

--- a/crates/webview/Cargo.toml
+++ b/crates/webview/Cargo.toml
@@ -12,9 +12,9 @@ repository = "https://github.com/longbridge/gpui-component"
 gpui.workspace = true
 gpui-component.workspace = true
 wef.workspace = true
+serde.workspace = true
+schemars.workspace = true
+raw-window-handle.workspace = true
+rust-i18n.workspace = true
 
 image = { version = "0.25.6", default-features = false }
-serde = { version = "1.0.219", features = ["derive"] }
-schemars = "0.8.22"
-raw-window-handle = "0.6.2"
-rust-i18n = "3.1.5"

